utils/create-module.js

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env node
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const os = require('os');
+
+/**
+ * Add the module as a dependency in modules/_canvas-kit-<TYPE>/package.json.
+ */
+const addModuleAsDependencyByType = (cwd, moduleName, type) => {
+  const packagePath = path.join(cwd, `modules/_canvas-kit-${type}/package.json`);
+  const packageContents = fs.readFileSync(packagePath);
+  const json = JSON.parse(packageContents);
+
+  // Add module, sort dependencies, then write back out
+  json.dependencies[`@workday/canvas-kit-${type}-${moduleName}`] = `^${json.version}`;
```

Comment:
Lmk if we want to switch this to that `0.0.0` one or if this will suffice

Reply:
Yes, please use `0.0.0`. We use Lerna to publish. The `0.0.0` will be bumped to the real version during this step. `0.0.0` solves the `yarn install` problem. `0.0.0` is treated special. A symlink will be created to `../../modules/${moduleName}/react` and everything will pass as expected.